autoagora/logs_db.py

```diff
@@ -52,7 +179,7 @@ async def get_most_frequent_queries(
                         Avg(query_time_ms) as avg_time,
                         Stddev(query_time_ms) as stddev_time 
                     FROM
-                        query_logs
+                        {query_logs_table}
```

Comment:
@aasseman So this part, didn't let me to either leave it as `%s` and insert values or using `$1`, since the syntax doesn't work if the variable is alone like that.
Had to add format to it and use it this way, however this is vulnerable and SQL injections this way are a possibility, although this variable is not accesible by the user, what do you think?
Should I then create a separate query so that we don't have that there?

Reply:
So looked further into this, the easiest quickest approach is to just separate the function into two or to have in the constanst file the query already defined and select the query depending the type of query its looking for , which should be the one that leaves the code the "cleanest"

Reply:
I did some research. I think the best we can do here is to use an f-string for the query to inject the right table identifier, **but** with wrapping the injected table identifier in the PG function `QUOTE_IDENT()` to make it safe.

Example here, but you can find way more on Google: https://dba.stackexchange.com/questions/5454/postgres-plpgsql-using-a-variable-inside-of-a-dynamic-create-statement/10864#10864